src/repositories/groups-repository.js

```diff
@@ -98,11 +98,20 @@ function handleId(queryParamId, response, Group, filter, models) {
   if (queryParamId) {
     const query = {
       ...filter,
-      raw: true,
       subQuery: false,
       ...buildIncludes(queryParamId, models),
     };
     response = Group.findAndCountAll(query);
+    return response.then((results) => {
+      const x = results.rows.map((item) => {
+        return {
+          id: item.id,
+          name: item.name,
+        };
+      });
+      results.rows = x;
+      return results;
+    });
   }
-  return response;
+  return Group.findAndCountAll();

```

Comment:
Again, if you always have the 'queryParamId' argument, this line will never be executed, and if it would happen to execute, you would receive everything, ignoring any filtering, limit and offset